index.html

```diff
@@ -1,12 +1,68 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="de">
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Tasse</title>
+    <link href="css" rel="stylesheet">
+    <scribt src="js" defer></scribt>
```

Comment:
```suggestion
    <scribt src="js" defer></scribt>
```
<script src="js" defer></script>